js/src/forum/components/RenameDiscussionModal.js

```diff
@@ -57,7 +57,7 @@ export default class RenameDiscussionModal extends Modal {
         .save({ title })
         .then(() => {
           if (app.viewingDiscussion(this.discussion)) {
-            app.current.stream.update();
+            app.current.streamState.update();
```

Comment:
Are we still storing the stream then?

Reply:
No, we're storing the state.

Reply:
Then why not just keep it as `app.current.stream`?

Reply:
Good point; we'll probably eventually move all the states into app.state or something with the current values acting as an alias, but till then, no benefit in unnecesssary breaks.